source/static/firmware/connect-zbt-1/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+    "product_name": "Home Assistant Connect ZBT-1",
+    "baudrates": {
+        "bootloader": [
+            115200
+        ],
+        "cpc": [
+            460800,
+            115200,
+            230400
+        ],
+        "ezsp": [
+            115200
+        ],
+        "spinel": [
+            460800
+        ]
+    },
+    "usb_filters": [
+        {
+            "pid": 60000,
+            "vid": 4292
+        }
+    ],
+    "firmwares": [
+        {
+            "name": "Zigbee (EZSP)",
+            "url": "/firmwares/skyconnect_zigbee_ncp_7.4.4.0.gbl",
```

Comment:
@puddly: where do the binaries live? Can I point to a source repo, rather than uploading them to the HA docs repo?

Reply:
GitHub unfortunately doesn't let you hotlink to release assets [due to CORS](https://github.com/orgs/community/discussions/45446). The files themselves are released from the firmware builder repo (here's the latest release https://github.com/NabuCasa/silabs-firmware-builder/releases/tag/v2025.04.04-1), I think we downloaded them into the ZBT-1 website assets and linked to them from there.

Reply:
correct, we did. but I did not remember the reason. So I thought I should probably add a link to the source repo instead. I will do the same here then, thx.